Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp

```diff
@@ -259,6 +260,16 @@ void MolDraw2DSVG::getStringSize(const std::string &label, double &label_width,
   }
 }
 
+namespace {
+void escape_xhtml(std::string &data) {
```

Comment:
I wasn't paying attention it was in a regression test: testFeatures

```
#include <boost/spirit/core.hpp>
```

So we would be adding a run-time dependency.